Add Remote Config changelog for PR #5753 (#5762)
firebase/firebase-android-sdk: firebase-config/CHANGELOG.md

# Unreleased

> * [fixed] Fixed a bug that could cause a crash when the app is backgrounded while listening for
>   real-time Remote Config updates. For more information, see
>   <a href="https://github.com/firebase/firebase-android-sdk/issues/5751"
>      class="external">GitHub Issue #5751</a>.

Reviewer: ```suggestion
* [fixed] Fixed a bug that could cause a crash if the app was backgrounded
  while it was listening for real-time Remote Config updates. For more information, see
  <a href="https://github.com/firebase/firebase-android-sdk/issues/5751"
     class="external">GitHub Issue #5751</a>.
```
